frontend/src/index.test.js

```diff
@@ -1,37 +1,51 @@
-import { render, screen } from "@testing-library/react";
-import {setupLandingPage} from "./setupTests";
+// TODO: Test cases for: makeGraph(), setDescendantStats(), getIndentedTreeNodes()?, cset comparison page line 144?
+// TODO: Inputs: real inputs in addition to wholeGraphEdge
```

Comment:
## TODO: Test inputs
I think in `test_graph.py`, can use some of those test cases. However I have to convert those cset IDs to the inputs needed for each function, e.g. querying its members_items.